components/interfaces/hybrid-email-interface.tsx

```
@@ -0,0 +1,368 @@
+import { Send, Inbox, Star, Archive, Trash2, Mail, RefreshCw, Bot, Zap } from "lucide-react"
+import type { gmail_v1 } from 'googleapis'
+import DOMPurify from 'isomorphic-dompurify'
+import { motion } from "framer-motion"
+
+// Render sanitized HTML to prevent injection
+const EmailBody = ({ htmlBody }: { htmlBody: string }) => {
+  const sanitized = useMemo(
+    () => DOMPurify.sanitize(htmlBody, { USE_PROFILES: { html: true } }),
+    [htmlBody]
+  )
+  return <div className="prose prose-invert max-w-none" dangerouslySetInnerHTML={{ __html: sanitized }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)